CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This release will bump the Realm file format from version 22 to 23. Opening a fi
 * Add support for `Realm.copyFromRealm()`. All RealmObjects, RealmResults, RealmList and RealmSets now also have a `copyFromRealm()` extension method.
 * [Sync] `App.close()` have been added so it is possible to close underlying ressources used by the app instance.
 * [Sync] Add support for progress listeners with `SyncSession.progress`. (Issue [#428](https://github.com/realm/realm-kotlin/issues/428))
+* Add better error messages when inheriting `RealmObject` with unsupported class types. (Issues [#1086](https://github.com/realm/realm-kotlin/issues/1086))
```

Comment:
this issue only mentions enum, so perhaps need to link to somewhere else? i.e. this PR?